src/v/cluster/controller_backend.cc

```diff
@@ -1078,13 +1079,15 @@ ss::future<result<ss::stop_iteration>> controller_backend::reconcile_ntp_step(
             // Configuration will be replicate to the new replica
             initial_replicas = {};
         }
-
         auto ec = co_await create_partition(
           ntp,
           group_id,
           expected_log_revision.value(),
           std::move(initial_replicas),
-          replicas_view.revisions());
+          replicas_view.revisions(),
+          force_reconfiguration(
+            replicas_view.update
+            && replicas_view.update->is_force_reconfiguration()));
```

Comment:
i was wondering if the recovery should be enabled only if all the replicas are new.. otherwise existing longest log replica that will be elected as a leader (after force reconfiguration) seeds the new replicas?